frontend/src/metabase/dashboard/actions/auto-wire-parameters/utils.ts

```diff
@@ -27,26 +30,26 @@ export function getAllDashboardCardsWithUnmappedParameters({
   dashboardId: DashboardId;
   parameterId: ParameterId;
   excludeDashcardIds?: DashCardId[];
-}) {
-  const cards = getExistingDashCards(
+}): QuestionDashboardCard[] {
+  const dashCards = getExistingDashCards(
     dashboardState.dashboards,
     dashboardState.dashcards,
     dashboardId,
   );
-  return cards.filter(dashcard => {
-    return (
+  return dashCards.filter(
+    dashcard =>
```

Comment:
We can replace the cast with this:

```suggestion
    (dashcard): dashcard is QuestionDashboardCard =>
```

Reply:
Fixed in [`1ed9a9a`](https://github.com/metabase/metabase/pull/38444/commits/1ed9a9a0172f0de410ad7885f5aab03cd193d0cc)